CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.5.2
+
+- Fix using full Entity Ref. This fixes an issue where the plugin was not resolving entity Kind and Namespace.
```

Comment:
Maybe you can help me then. With my own words: we were constructing links within Backstage based on `tag`/`codeTag` from entities (despite hiding those under misleading `componentRef` field, eh 😢  ). This was causing incorrect navigation as we were doing fallback to default `kind: Component` and loosing proper `kind`.

Reply:
OK wouldn't it be something like this
```suggestion
- Fix entity links by applying the correct `kind` instead of always assuming they are `Component`s
```
?